Corona-Warn-App/src/main/java/de/rki/coronawarnapp/service/diagnosiskey/DiagnosisKeyConstants.kt

```diff
@@ -60,24 +60,17 @@ object DiagnosisKeyConstants {
     private val PARAMETERS_DOWNLOAD_URL = "$VERSIONED_DISTRIBUTION_CDN_URL/$PARAMETERS"
     private val APPCONFIG_DOWNLOAD_URL = "$VERSIONED_DISTRIBUTION_CDN_URL/$APPCONFIG"
 
-    /** Index Download URL built from CDN URL's and REST resources */
-    val INDEX_DOWNLOAD_URL = "$VERSIONED_DISTRIBUTION_CDN_URL/$INDEX_FILE_NAME"
-
     /** Diagnosis key Download URL built from CDN URL's and REST resources */
     val DIAGNOSIS_KEYS_DOWNLOAD_URL = "$VERSIONED_DISTRIBUTION_CDN_URL/$DIAGNOSIS_KEYS"
 
     /** Diagnosis key Submission URL built from CDN URL's and REST resources */
     val DIAGNOSIS_KEYS_SUBMISSION_URL = "$VERSIONED_SUBMISSION_CDN_URL/$DIAGNOSIS_KEYS"
 
-    /** Country-Specific Parameter URL built from CDN URL's and REST resources */
```

Comment:
I think that comment on line 72 not only applied to line 73, but also to line 74 (maybe even line 76).

Reply:
You're right. I've added the comment back in.